docs/NetworkZoneApi.md

```diff
@@ -86,9 +86,9 @@ Name | Type | Description  | Notes
 | Status code | Description | Response headers |
 |-------------|-------------|------------------|
 | **200** | Success |  -  |
-| **403** | Forbidden |  -  |
-| **404** | Not Found |  -  |
-| **429** | Too Many Requests |  -  |
```

Comment:
Was it intentional that the descriptions were removed?  It seems preferable to keep them.

Reply:
It all depends on the spec, the code is autogenerated.

Reply:
I thought that was the case; should we review the spec more closely?